main.go

```diff
@@ -2857,6 +2893,45 @@ prints out information about a certificate, including:
 			}
 			fmt.Printf("\n")
 
+			fmt.Printf("  signed with the algorithm ")
+			var sigAlgo string
+			switch cert.Certificate.SignatureAlgorithm {
```

Comment:
Move this switch into x509 as GetSignatureLable (or similar) -- giant case statement doesn't belong in what is essentially a View

Reply:
Actually took this style from the key usage stuff right above it.